packages/commonwealth/client/scripts/views/pages/view_snapshot_proposal/index.tsx

```diff
@@ -183,4 +214,4 @@ export const ViewSnapshotProposalPage = ({
   );
 };
 
-export default ViewSnapshotProposalPage;
+export default ViewProposalPage;
```

Comment:
this file name is view snapshot proposal, so why it is changed to view proposal page?